charts/brigade-acr-gateway/values.yaml

```diff
@@ -0,0 +1,129 @@
+    ## 2. OR Set values for BOTH the cert and key fields below
+    ## 3. OR create a cert secret named
+    ##    <Helm release name>-ingress-cert in the same namespace as the gateway.
+    ##    This secret could be created manually or through other means, such as
+    ##    a cert manager.
+    ##
+    ## Note there is a wide disparity in the feature set of various ingress
+    ## controllers and some ingress controllers may be able to provision a
+    ## certificate for you even with TLS disabled here. Consult your ingress
+    ## controller's documentation.
+    enabled: true
+    ## Whether to generate a self-signed certificate. If true, a new
+    ## certificate will be generated for every revision of the corresponding
+    ## Helm release. Since the certificate is self-signed, it will not be
+    ## trusted by clients and should absolutely not be used for production,
+    ## but having this enabled as a default effectively discourages the more
+    ## heavy-handed option to disable TLS entirely. If ingress TLS is enabled
+    ## and cert generation is DISABLED, users MUST provide their own cert and
+    ## private key below OR create a cert secret named
+    ## <Helm release name>-ingres-cert in the same namespace as the gateway.
+    generateSelfSignedCert: true
+    # cert: base 64 encoded cert goes here
+    # key: base 64 encoded key goes here
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as
+  # a conscious choice for the user. This also increases chances charts run on
+  # environments with little resources, such as Minikube. If you do want to
+  # specify resources, uncomment the following lines, adjust them as
+  # necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+service:
+  ## If you're going to use an ingress controller, you can change the service
+  ## type to CLusterIP.
+  type: LoadBalancer
+  # nodePort: 31700
+
+brigade:
+  ## Address of your Brigade 2 API server, including leading protocol (http://
+  ## or https://)
+  apiAddress: https://brigade-apiserver.brigade.svc.cluster.local
+  ## API token belonging to a Brigade 2 service account
+  ##
+  ## For reference, these are the commands for creating a service account for
+  ## this gateway, with proper permissions:
+  ##
+  ## $ brig service-account create --id brigade-acr-gateway --description brigade-acr-gateway
+  ## $ brig role grant READER --service-account brigade-acr-gateway
+  ## $ brig role grant EVENT_CREATOR --service-account brigade-acr-gateway --source brigade.sh/acr
+  apiToken: 7eb00823d0d349beba85c4d22dfea1baWPOC7Fh1PFNtZyi8b7ZYMxy7VglKunpsx6i5VZd6MSlX5O48b62by3IQxUdBm9VABAFFPgylGYuS9R2MzMkIxC7R3C9DSIaE2udZF5bxpqIlDAIqnvrVqy6yLzY8Wipa4foFL3wasISIt3ZMuPgntfSJrE17iCnNIHW1BR6flzVzUZDpG1zyAWKEDiTauqE7iczhkflGlfDbwjvRnDp4vXpKcyMrNL5h
```

Comment:
```suggestion
  apiToken:
```

Reply:
I did it again, huh? Will fix, but as usual... no exposure. That token was for an instance of the gateway that was running locally and has since been deprovisioned.